javascript-functions-practice/get-character-at-index.js

```diff
@@ -0,0 +1,5 @@
+/* exported getCharacterAtIndex */
+function getCharacterAtIndex(string, index) {
+  var CharacterAtIndex = string[index];
```

Comment:
we do not want to use Capital letters for variable names except in special circumstances of which this is not one :)